diktat-rules/src/test/resources/test/chapter6/init_blocks/InitBlocksTest.kt

```diff
@@ -0,0 +1,9 @@
+package test.chapter6.init_blocks
+
+class Example {
+    init { println("Lorem ipsum") }
```

Comment:
@petertrr I have started thinking right now that there is a possibility to break the code here. 
If someone will do the following:
```
init { println(1) }
    val a = "".also {
        println(3)
    } 
init { println(2) }
```

We will break expected print chain